src/header.rs

```diff
@@ -237,7 +237,7 @@ impl NiftiHeader {
     /// Currently, only the following problems are fixed:
     /// - If `pixdim[0]` isn't equal to -1.0 or 1.0, it will be set to 1.0
     pub fn fix(&mut self) {
-        if self.pixdim[0].abs() != 1.0 {
+        if (self.pixdim[0].abs() - 1.).abs() > 1e-11 {
```

Comment:
I see that we use this comparison 2 times. It's nothing dramatic of course, but we might want to add a `is_pixdim_valid` method or something like that.

Reply:
I moved it to a separate function now.